### ISSUE TITLE
Feature request: Policy that blocks access to about:policies

### ISSUE THREAD

**Issue by drlellinger** [mozilla/policy-templates#631]
I know that seems pretty harsh but i could imagine a usecase where companies do not want to disclose the forbidden sites to their employees or customers.

**Comment by mkaply** [comment]
That would sure make problems difficult to debug...

**Comment by drlellinger** [comment]
Yes i have already thought about that that's why we do not want to enable the policy now but maybe in the future.
